src/rules/indentation.js

```diff
@@ -42,7 +47,9 @@ function indentation(parsedFile, unused, configuration) {
 
     scenario.steps.forEach(function(step) {
       // Check Step indentation
-      test(step.location, configuration, 'Step');
+      var keyword = step.keyword.trim();
+      var type = keyword in availableConfigs ? keyword : 'Step';
```

Comment:
I've added this check to be "future" proof, in case Gherkin decides to add e.g. a new step keyword like *Anyways*.
Even more, the keywords are localized, see e.g. https://github.com/cucumber/gherkin/blob/master/gherkin-languages.json
That means e.g. the German localized version of `But` is `Aber`, which won't be in the availableConfigs.

I can change the check to `keyword in configuration`, but the we have to keep in mind that https://github.com/vsiakka/gherkin-lint/blob/master/src/rules.js#L65 does check if the given configuration option is in the availableConfig options, maybe this should be disabled for steps, since they are very dynamic due to different languages.